lib/topic_view.rb

```diff
@@ -762,8 +783,16 @@ def unfiltered_posts
     result
   end
 
-  def apply_default_order(scope)
-    scope.order(sort_order: :asc)
+  def apply_default_scope(scope)
+     scope = scope.order(sort_order: :asc)
+
+     if self.class.custom_default_scopes.present?
```

Comment:
Nitpick, but I don't think this check is necessary, because you always re-initialize custom_default_scopes if it is null when you call it:

```
def self.custom_default_scopes
    @custom_default_scopes ||= []
  end
```

Reply:
O good catch :+1: